### PR TITLE
Add email social sharing provider

### PR DESCRIPTION
**Pull request by rullzer**

### CODE REVIEW

File: socialsharing_email/js/socialsharingemail.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+(function() {
+	var email = new OC.Share.Social.Model({
+		key: 'email',
+		url: 'mailto:?subject=I shared a file with yout&body={{reference}}',
```
Comment (schiessle) [inline socialsharing_email/js/socialsharingemail.js]: I would suggest to translate the subject

Comment (rullzer) [inline socialsharing_email/js/socialsharingemail.js]: Mmm fair enough but then we have to setup transifex for this app. I suggest to get it in now. And fix it later ;)

Comment (schiessle) [inline socialsharing_email/js/socialsharingemail.js]: OK, let's fix this later

**Comment by schiessle** [comment]
I just tried it. When I click on the "share to mail" button my mail client gets started but instead of a mail composer it open a empty browser window along with the mail client...

**Comment by rullzer** [comment]
mmmm I think that is the server part (that actually opens a window)... maybe we need some magic there... It might also be browser dependant
anwyays will look into it.

**Pull request merged**